tests/test_ReliabilityProblem110.py

```diff
@@ -13,22 +13,21 @@
 
```

Comment:
Fixing the style of the unit tests is good. However, I guess that a separate PR would have made this review clearer. In this PR, it hides the fact that there is no unit test for the new classes, letting the reader think that there are unit tests, while there are not. Please avoid this for future PRs. Anyway, keep them as is for this one to smooth the process.